docs/user-guide/jobs.md

```diff
@@ -0,0 +1,256 @@
+```
+
+## Writing a Job Spec
+
+As with all other Kubernetes config, a Job needs `apiVersion`, `kind`, and `metadata` fields.  For
+general information about working with config files, see [here](simple-yaml.md),
+[here](configuring-containers.md), and [here](working-with-resources.md).
+
+A Job also needs a [`.spec` section](../devel/api-conventions.md#spec-and-status).
+
+### Pod Template
+
+The `.spec.template` is the only required field of the `.spec`.
+
+The `.spec.template` is a [pod template](replication-controller.md#pod-template).
+ It has exactly the same schema as a [pod](pods.md), except
+it is nested and does not have an `apiVersion` or `kind`.
+
+In addition to required fields for a Pod, a pod template in a job must specify appropriate
+lables (see [pod selector](#pod-selector) and an appropriate restart policy.
+
+Only a [`RestartPolicy`](pod-states.md) equal to `Never` or `OnFailure` are allowed.
+
+### Pod Selector
+
+The `.spec.selector` field is a pod selector.  It works the same as the `.spec.selector` of
+a [ReplicationController](replication-controller.md).
+
+If specified, the `.spec.template.metadata.labels` must be equal to the `.spec.selector`.  Config not following
+this rule will be rejected by the API.  If unspecified, it will be defaulted to the latter.
+
+Also you should not normally create any pods whose labels match this selector, either directly,
+via another Job, or via another controller such as ReplicationController.  Otherwise, the Job will
+think that those pods were created by it.  Kubernetes will not stop you from doing this.
+
+### Multiple Completions
+
+By default, a Job is complete when one Pod runs to successful completion.  You can also specify that
+this needs to happen multiple times by specifying .spec.completions` with a value greater than 1.
```

Comment:
I'm reading your statement as: valid values are 1 and above, whereas our validation allows 0 and above, thus the question is should we leave validation as is for completions or change it to be 1 and above, because it doesn't make sense to have 0 completions does it?